index.html

```diff
@@ -1,20 +1,207 @@
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+    <!-- JavaScript -->
+    <link rel="stylesheet" href="javascript/javascript.js">
+    <title>Responsive grid project</title>
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+    <header class="container">
+        <div class="row justify-content-around">
+            <ul class="col-9 offset-1">
+                <li class="d-inline"><a href="https://twitter.com/wix"><img
+                            src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/c7d035ba85f6486680c2facedecdcf4d.webp"
+                            alt="Twitter icon"></a></li>
+                <li class="d-inline"><a href="https://www.youtube.com/user/Wix"><img
+                            src="https://static.wixstatic.com/media/78aa2057f0cb42fbbaffcbc36280a64a.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/78aa2057f0cb42fbbaffcbc36280a64a.webp"
+                            alt="YouTube icon"></a></li>
+                <li class="d-inline"><a href="https://www.facebook.com/wix"><img
+                            src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/0fdef751204647a3bbd7eaa2827ed4f9.webp"
+                            alt="Facebook icon"></a></li>
+                <li class="d-inline"><a href="https://www.instagram.com/wix/"><img
+                            src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.webp"
+                            alt="Instagram icon"></a></li>
+                <li class="d-inline"><a href="https://soundcloud.com/"><img
+                            src="https://static.wixstatic.com/media/03ea2af965014a3eb32df628a4c18c0e.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/03ea2af965014a3eb32df628a4c18c0e.webp"
+                            alt="SoundCloud icon"></a></li>
+            </ul>
+            <div class="col-1">
+                <a href=""><img src="img/menu-hamburger.svg" alt="hamburger menu"></a>
+            </div>
+        </div>
+
+
+
+
+
+
+
+
+
+    </header>
+
+    <main>
+        <section class="container-fluid hero-image">
+            <div class="row justify-content-center">
+                <div class="col-6 offset-3">
+                    <h1>DEW FESTIVAL MAY 28</h1>
+                    <h2>May 28, 2023, 9:00 AM</h2>
+
+                    <button class="ticket-button" type="button" aria-placeholder="Buy Tickets">Buy
+                        Tickets</button>
+                </div>
+        </section>
+        <section class="container-fluid line-up-section">
+            <div class="row">
+                <div class="col-10 offset-1 col-md-8 offset-md-2">
```

Comment:
Here, in order to separate the background shapes and be able to control them (to rotate them and size them appropriately), I suggest moving the background-images to the column level, and having 2 separate column blocks: one for the backgrounds, and one for the text. This way, you can rotate, size, style as you wish, without affecting the others:
```
    <section class="container-fluid line-up-section">
      <div class="row">
        <div class="background-shapes col-10 offset-1 col-md-8 offset-md-2"></div>
        <div class="content col-10 offset-1 col-md-8 offset-md-2">
          <h3>Line up</h3>
          <p>ANDRE'S BEST / BLAKE ADAMSON / CAMMILLE / DJ CANOPY / DELORA PLANK / BELL HARRIS / NADINE BELMONT etc. </p>
        </div>
      </div>
    </section>
```
The `background-shapes` class can deal with the background-shapes, and also apply `transform: rotate(135g)` to match the design.
You'll notice if you follow this, the shape will be one block, and the text, a separate block below. To remedy this, apply `position: absolute` to the `background-shapes` class, as well as a `z-index` to both blocks to define which one sits on top.

Reply:
One more hint: `background-shapes` will need a `height` set for it to appear, as it will not have content of its own to grow the block.